src/CommandHandler.ts

```diff
@@ -241,7 +241,16 @@ export namespace CommandHandler {
 		await client.login(options.token);
 		prefixes.push(`<@${client?.user?.id}> `);
 		prefixes.push(`<@!${client?.user?.id}> `);
-		owners.push((await client.fetchApplication()).owner?.id ?? '');
+		const appOwner = (await client.fetchApplication()).owner
+		if (appOwner) {
+			if (appOwner instanceof Team) {
+				for(const member of appOwner.members.map(m => m)) {
+					owners.push(member.id)
+				}
+			} else {
+				owners.push(appOwner.id)
+			}
+		}
```

Comment:
Remember to use the `prettier` script to prettify these lines to accord to the code format.

Reply:
Forgot about that